packages/calcite-components/src/components/handle/handle.scss

```diff
@@ -19,17 +35,20 @@
   }
 }
 
-:host(:not([disabled])) .handle {
+:host(:hover) .handle {
```

Comment:
I have to leave .hover here, otherwise, it stretches all across the available width.